dev/preview/infrastructure/modules/gce/vm.tf

```diff
@@ -14,6 +14,7 @@ resource "google_compute_instance" "default" {
     initialize_params {
       image = "projects/workspace-clusters/global/images/${var.vm_image}"
       type  = "pd-ssd"
+      size  = 512
```

Comment:
I just built a preview from main, here is what I see:
```
ubuntu@kylos101-a08xuppoicg:~$ df -h
Filesystem                                Size  Used Avail Use% Mounted on
tmpfs                                     6.3G   15M  6.3G   1% /run
/dev/sda5                                  30G  6.8G   24G  23% /
tmpfs                                      32G     0   32G   0% /dev/shm
tmpfs                                     5.0M     0  5.0M   0% /run/lock
/dev/mapper/lvm--disk-ipfs                 12M 1004K   11M   9% /var/gitpod/ipfs
/dev/mapper/lvm--disk-gitpod--workspaces   10G  105M  9.9G   2% /var/gitpod/workspaces
/dev/mapper/lvm--disk-containerd--mount    49G   17G   33G  34% /var/lib/containerd
/dev/sda2                                  99M  6.1M   93M   7% /boot/efi
```

The disk is 100GB:
![image](https://github.com/gitpod-io/gitpod/assets/1272076/768e299c-062c-4638-bf43-f0538c8a3d4b)

If we bump this to 200GB, or 512GB, how do you expect this to change? I consider this a blocking question.

It looks like the kubelet has 30GB accessible, 24GB available:
```
root@kylos101-a08xuppoicg:/var/lib/kubelet# df -h .
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda5        30G  6.8G   24G  23% /
```

Reply:
setting the disk size to 512GB increases the available ephemeral storage to 442GB:
```
root@wv-cache-agent-build:/var/lib/kubelet# df -h .
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda5       442G  9.7G  433G   3% /
```
(the node also reports this much allocatable `ephemeral-storage`, from `kubectl describe node`)

As discussed I'll change it from 512 to 256GB

Reply:
Updated